variables.tf

```diff
@@ -3,8 +3,21 @@
 ##############################################################################
 
 variable "name" {
-  description = "Name for VPC"
+  description = "Name for VPC."
   type        = string
+  default     = null
```

Comment:
hmm, by setting the default value to null, it means the current set of the module variable default values won't work out of the box (since `create_vpc` is `true` by default, meaning `name` is required). Perhaps we default the value to some real string like "dev" ?